cms/static/sass/amc-specific.scss

```diff
@@ -144,3 +144,31 @@ body.view-settings {
     }
   }
 }
+
+body {
+  /*
+   * RED-1504: Hide the checklists feature because it's not designed to work for Tahoe customers
```

Comment:
The actual change, since `amc-specific.css` is a compiled version of the SCSS file.